components/brave_ads/core/internal/ml/pipeline/text_processing/embedding_processing_unittest.cc

```diff
@@ -40,21 +40,20 @@ TEST_F(BatAdsEmbeddingProcessingTest, EmbedText) {
       resource.Get();
   ASSERT_TRUE(embedding_processing);
 
-  const std::vector<std::tuple<std::string, ml::VectorData>> samples = {
-      {"this simple unittest", ml::VectorData({0.5, 0.4, 1.0})},
-      {"this is a simple unittest", ml::VectorData({0.5, 0.4, 1.0})},
-      {"this is @ #1a simple unittest", ml::VectorData({0.5, 0.4, 1.0})},
-      {"that is a test", ml::VectorData({0.0, 0.0, 0.0})},
-      {"this 54 is simple", ml::VectorData({0.85, 0.2, 1.0})},
+  const std::vector<std::tuple<std::string, std::vector<float>>> k_samples = {
```

Comment:
invalid naming `k_samples`

Reply:
if I try `kSamples` clangd is telling me `invalid case style`

Reply:
Please reach out to @aseren and @tmancey in slack DM